.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug Report
+description: 'Use this template for tracking new bugs. Make sure to add the severity in the issue title (Low, Medium or High)title: "BUG: [SEVERITY]: [BUG DESCRIPTION]"'
+title: "BUG: [SEVERITY]: [BUG DESCRIPTION]"
+labels: [bug]
+body:
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: Provide a clear description of the bug you encoutered.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Repro
+      description: List the steps needed to reproduce the bug. Be as clear and detailed as possible.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: Descibe what should be done for this bug to be considered as fixed. 
```

Comment:
instead of what should be done I'd have this section describe the expected behavior in contrast to the behaviour caused by the bug

Reply:
eg 
bug desc: price shown in uIST on buy flow
repro: browse character market
expected behaviour: price shown in IST